bundles/org.openhab.binding.wemo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -318,6 +350,10 @@
 		<item-type>String</item-type>
 		<label>Air Quality</label>
 		<description>Shows the air quality measured by a WeMo enabled Holmes Air Purifier</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Mode</tag>
```

Comment:
Seems more like a measurement of air quality, not sure what appropriate tags are.

Reply:
Applying `Measurement.AirQuality`